utils/webhook/github.handler.ts

```diff
@@ -0,0 +1,451 @@
+import prisma from "../../prisma";
+import { createHmac, timingSafeEqual } from "crypto";
+import {
+    decrypt,
+    formatJSON,
+    getAttachmentQuery,
+    replaceImgTags,
+    skipReason
+} from "../index";
+import { LinearClient } from "@linear/sdk";
+import { replaceMentions, upsertUser } from "../../pages/api/utils";
+import { IssueCommentCreatedEvent, IssuesEvent } from "@octokit/webhooks-types";
+import { generateLinearUUID } from "../linear";
+import { LINEAR } from "../constants";
+import petitio from "petitio";
+import { linearQuery } from "../apollo";
+import { ApiError } from "../errors";
+
+export async function githubWebhookHandler(

```

Comment:
Contents of this function are the contents of the `github` if statement previously

Reply:
Have you replaced early `res.send()` calls with `return`s / `throw`s?